cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxActivity.java

```diff
@@ -301,13 +304,15 @@ protected void onCreate(final Bundle savedInstanceState) {
     protected void onResume() {
     	Log.d(TAG, "onResume()");
         super.onResume();
+        this.hideVirtualButton();
        	resumeIfHasFocus();
     }
     
     @Override
     public void onWindowFocusChanged(boolean hasFocus) {
     	Log.d(TAG, "onWindowFocusChanged() hasFocus=" + hasFocus);
         super.onWindowFocusChanged(hasFocus);
+        this.hideVirtualButton();
```

Comment:
Should we check `hasFocus` value? hideVirtualButton need to be invoked only while hasFocus is `true`.